### DISCUSSION TITLE
What is a best practice to share settings between SBT and the running application?

### DISCUSSION THREAD

**Discussion by kincsescsaba** [sbt/sbt#7611] [Q&A] [answered]
Hey there,

I'd like to know how to accomplish something like loading the same `.conf` or `.properties` file in SBT and my app to determine, for example, the exact location of the generated jars.

Any ideas?

**Comment by eed3si9n** [comment, marked as answer]
sbt-buildinfo maybe?

**Reply by kincsescsaba** [reply]
Yes, I think I'm looking for a tool like this, thanks
